charts/pdblib/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: pdb-lib
+description: A libary Helm chart for Pod Disruption Budgets
+type: library
+version: 0.0.0
```

Comment:
Do we usually start at `0.0.0` or `0.0.1`?

Reply:
I started with 0.0.0 because the chart releaser will bump it to 0.1.0